autohaven/autohaven_app/migrations/0012_seller_alter_listing_mileage.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.1 on 2024-07-16 22:02
```

Comment:
Please remove this file from this PR. This migration is not needed

Reply:
@Juanpam I have fixed the issues can I resolve the conversation?